Promise.reject('rejected for some reason')